src/MeetingsSDKAdapter.test.js

```diff
@@ -639,6 +639,12 @@ describe('Meetings SDK Adapter', () => {
   });
 
   describe('shareControl()', () => {
+    beforeEach(() => {
+      meetingSDKAdapter.meetings[meetingID] = {
+        ...meeting,
+        localShare: {},
```

Comment:
why is this needed?

Reply:
otherwise it doesn't know about the localShare property